### PR TITLE
Support downloading (exporting) the device configuration

### PR DESCRIPTION
**Pull request by saimonation**

### CODE REVIEW

File: cterasdk/edge/config.py [hunk 1]
```diff
@@ -42,6 +45,15 @@ def set_hostname(self, hostname):
         logging.getLogger().info('Configuring device hostname. %s', {'hostname': hostname})
         return self._gateway.put('/config/device/hostname', hostname)
 
+    def export(self):
```
Comment (ygalblum) [inline cterasdk/edge/config.py:48]: I think you should add an optional file destination path

Comment (saimonation) [reply cterasdk/edge/config.py]: Done

File: cterasdk/edge/config.py [hunk 2]
```diff
@@ -42,6 +48,26 @@ def set_hostname(self, hostname):
...
+        :param str,optional destination:
+         File destination, defaults to the default directory
+        """
+        default_filename = self._gateway.host() + datetime.now().strftime('_%Y-%m-%dT%H_%M_%S') + '.xml'
+        directory = filename = None
+        if destination:
+            directory, filename = self._filesystem.split_file_directory(destination)
+        else:
+            directory = self._filesystem.get_dirpath()
+            filename = default_filename
```
Comment (ygalblum) [inline cterasdk/edge/config.py:63]: If the code reaches here (line 63), the if statement in line 64 will always be false. So, either remove this line or move the if statement into the previous if scope

Comment (saimonation) [reply cterasdk/edge/config.py:65]: Funny you caught that. I immediately fixed after I pushed my changes yesterday but held off pushing again. Done. I added it to the previous scope.

Comment (ygalblum) [reply cterasdk/edge/config.py:65]: 👍

**Pull request merged**